src/test/java/com/github/minecraftschurlimods/arsmagicalegacy/test/BossSpawnTest.java

```diff
@@ -205,7 +205,7 @@ public static void testLifeGuardianSpawnRitual(GameTestHelper helper) {
         });
     }
 
-    @GameTest(template = "arcane_guardian_spawn_test", batch = BATCH, timeoutTicks = 20, attempts = 2)
+    @GameTest(template = "arcane_guardian_spawn_test", batch = BATCH, timeoutTicks = 20, required = false)
```

Comment:
Add a comment saying why it is required = false and a todo to make this a reliable test